utility_analysis_new/combiners.py

```diff
@@ -549,7 +520,13 @@ def merge_accumulators(self, acc1: AccumulatorType, acc2: AccumulatorType):
     def compute_metrics(self,
                         acc: AccumulatorType) -> metrics.AggregateErrorMetrics:
         """Computes metrics based on the accumulator properties."""
+        return self._compute_metrics(acc, metrics.AggregateMetricType.COUNT)
+
+    def _compute_metrics(self, acc: AccumulatorType,
```

Comment:
Because we need to specify the `metric_type` differently for PrivacyIdCountCombiner and for CountCombiner. Is there a more Python-idiomatic way of doing this?

If `compute_metrics()` takes as input `metric_type`, I think the pipeline would fail because the collection doesn't have that.

Reply:
Thanks for explanation!